### ISSUE TITLE
Using the GPRS_SIM900 not initialize

### ISSUE THREAD

**Issue by masalinas** [Seeed-Studio/Seeeduino_GPRS#6]
I have a seeeduino GPRS. I'm using the library GPRS_SIM900 like you recomend from the wiki. Deploy the GPRS_TCPConnection.ino example, change the TX, RX pins also:
# define PIN_TX 8
# define PIN_RX 7

But when deploy I obtain this error from my Serial debug:

init error
init error
....

This library GPRS_SIM900 is really compatible with Seeeduino GPRS??

Regards

**Comment by lanselambor** [comment]
Thank you Masalinas, the problem is Seeeduino GPRS need to be powered up  by pin D9.

```
void SIM800::preInit(void)
{
    pinMode(SIM800_POWER_STATUS,INPUT);
    delay(10);
    if(LOW == digitalRead(SIM800_POWER_STATUS))
    {
        if(sendATTest() != 0)
        {
            delay(800);
            digitalWrite(powerPin,HIGH);
            delay(200);
            digitalWrite(powerPin,LOW);
            delay(2000);
            digitalWrite(powerPin,HIGH);
            delay(3000);  
        }
        while(sendATTest() != 0);                
        //Serial.println("Init O.K!");         
    }
    else
    {
        Serial.println("Power check failed!");  
    }
} 
```

**Comment by tstanescu1** [comment]
Hi, sorry to re-open such an old ticket. I'm having a similar issue with the init. I've set the D9 pin to HIGH, but the issue persists.

For some reason it only intermittently recognizes the actual modem, and most of the time I get an "init failed"... Not sure what's going on? Can it be the power? I've tried several power sources and I get the same issue? The power switch on the board is set to 5V. 

The weirdest thing is that the NET light is flashing constantly.

Thank you for your help
